detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/UIModuleIdlingResource.java

```diff
@@ -122,6 +140,33 @@ protected boolean checkIdle() {
         return true;
     }
 
+    // This is a workaround for https://github.com/facebook/react-native/issues/32594
+    // uses duck typing heuristics to determine that this is probably the stuck Switch operation and if so, ignores it
+    private boolean workaroundForRN66Bug(Object uiOperationQueue) {
+        boolean isStuckSwitchOperation = false;
+
+        try {
+            Class<?> DispatchCommandViewOperation = Class.forName(CLASS_DISPATCH_COMMAND_VIEW_OPERATION);
+            List<?> mViewCommandOperations = Reflect.on(uiOperationQueue).field(FIELD_MVIEW_COMMAND_OPERATIONS).get();
+
+            if (mViewCommandOperations.size() == 1) {
+                Class<?> ReadableArray = Class.forName(CLASS_READABLE_ARRAY);
+                Object viewOperation = DispatchCommandViewOperation.cast(mViewCommandOperations.get(0));
```

Comment:
Could you verify that it is a `DispatchStringCommandOperation` class, specifically?